src/pages/HomePage.tsx

```diff
@@ -0,0 +1,5 @@
+export const HomePage = () => (
+  <h1 className="title">
+    Home page
+  </h1>
```

Comment:
Wrap the page with container div from App, and others as well
```suggestion
    <div className="container">
    <h1 className="title">Home page</h1>
  </div>
```